### PR TITLE
release announcement for ripasso 0.6.0

### PR DESCRIPTION
**Pull request by alexanderkjall**

### CODE REVIEW

File: _posts/2022-11-20-Ripasso-release-0-6-0.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Release of Ripasso version 0.6.0"
+author: capitol
+category: infrastructure
+---
+![ripasso-cursive](/images/ripasso-cursive-0.4.0.png)
+
+Time passes as water under a bridge, it is yet again to release a version of ripasso. We present version 0.6.0.
```
Comment (fnords) [inline _posts/2022-11-20-Ripasso-release-0-6-0.md:9]: perhaps it should be "it is yet again time to"

File: _posts/2022-11-20-Ripasso-release-0-6-0.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+The sequoia implementation doesn't support reading the gpg keyring on the system, so if that
+one in chosen all public pgp keys must be imported imported into ripasso. But it can talk
+to the gpg-agent, so it doesn't require that the private key is imported into sequoia.
+
+#### Support for TOTP codes
+
+Ripasso now supports otpauth urls, if there is a url on the format `otpauth://` then a MFA token
+can be copied with `ctrl-b`.
+
+#### Support the wayland copy buffer
```
Comment (fnords) [inline _posts/2022-11-20-Ripasso-release-0-6-0.md:36]: should wayland be capitalized?

File: _posts/2022-11-20-Ripasso-release-0-6-0.md (hunk 1 above)
Comment (alexanderkjall) [reply _posts/2022-11-20-Ripasso-release-0-6-0.md]: fixed

File: _posts/2022-11-20-Ripasso-release-0-6-0.md (hunk 2 above)
Comment (alexanderkjall) [reply _posts/2022-11-20-Ripasso-release-0-6-0.md]: fixed

**Review by fnords — APPROVED** [review]

**Pull request merged**